job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/BuildParametersContext.groovy

```diff
@@ -417,4 +438,13 @@ class BuildParametersContext extends AbstractExtensibleContext {
         }
         node
     }
+
+    /**
+     * @since 1.47
+     */
+    @PackageScope
+    Item getItem() {
```

Comment:
It will return `super.item`. I wouldn't normally add it, but I have seen such method added in `StepContext` and thought you had some kind of convention in place for classes extending `AbstractExtensibleContext`. Should I just delete it?